Aquarius/Src/Aquarius/Renderer/Renderer.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include "Aquarius/Renderer/VertexArray.h"
+#include "Aquarius/Renderer/IndexBuffer.h"
+#include "Aquarius/Renderer/Shader.h"
+
+#include <glm/glm.hpp>
+
+
+namespace Aquarius {
+
+    class Renderer
+    {
+    public:
+        Renderer() = delete;
+        ~Renderer() = default;
+
+        static void Init();
+        static void Shutdown();
+
+        static void Clear();
+        static void ClearColor(glm::vec3 color);
+
+        static void Submit(VertexArray* vertexArray, Shader* shader);
+    };
```

Comment:
So will objects of this class ever need to be instantiated? If so, wdyt about moving these methods into a `Renderer` namespace instead? Seems like there is no particular gain to these being in a class vs namespace as the default constructor is deleted, there are no data members, and every method is static

Reply:
Though if the plan is to extend this and add functionality that would require us instantiating the object then disregard the above haha

Reply:
Yeah there's a concept called instanced rendering, although I doubt we will ever implement that. This renderer class will have static members only which can also just be defined in a namespace, so that sounds good.